package.json

```diff
@@ -51,14 +51,16 @@
   },
   "devDependencies": {
     "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.3.2",
-    "@testing-library/user-event": "^7.1.2",
+    "@testing-library/react": "^12.1.2",
+    "@testing-library/user-event": "^13.5.0",
     "autoprefixer": "^10.4.1",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-prettier": "^4.0.0",
     "lodash.debounce": "^4.0.8",
+    "msw": "^0.36.7",
     "postcss": "^8.4.5",
-    "prettier": "^2.5.1"
+    "prettier": "^2.5.1",
+    "react-test-renderer": "^17.0.2"
```

Comment:
NIT: I would lock the dependency versions. If you decide to do it, do it in a separate PR.

Reply:
Ok, will resolve with another PR